### ISSUE TITLE
babel preset-stage-2 requirement?

### ISSUE THREAD

**Issue by JeffBeltran** [shayneo/vue-fuse#28]
I didn't see it listed in the instructions so wanted to post and see if this is the correct way to solve the issue i ran into.

Following the setup instructions on the site i got
```
ERROR  Failed to compile with 1 errors 

 error  in ./node_modules/vue-fuse/src/VueFuse.vue

Module build failed: Error: Couldn't find preset "stage-2" relative to directory
```

this was solved by adding `npm install --save-dev babel-preset-stage-2`

Is this expected behavior? or am i doing something wrong?

Thanks in advance,
Jeff

**Comment by shayneo** [comment]
1. Thank you so much for reporting/cataloging the fix here. 🎉 
2. It seems like a rogue dependency may have slipped into the package somehow, I'll be doing some troubleshooting so that (hopefully) others won't have to jump through this hoop in later releases.

**Comment by iEmrul** [comment]
The issue still persists as of Today, and @JeffBeltran solution worked. Thanks

**Comment by shayneo** [comment]
Should be resolved in 2.0.0 release
